build.gradle

```diff
@@ -7,38 +7,56 @@
 // PLUGINS
 // -----------------------------------------------------------------------------
 
+buildscript {
+    repositories {
+        maven {
+            url 'https://plugins.gradle.org/m2/'
+        }
+    }
+    dependencies {
+        classpath 'io.github.gradle-nexus:publish-plugin:1.3.0'
+    }
+}
```

Comment:
Why are we using the old method of plugins rather than just adding:

```
plugins {
    id("io.github.gradle-nexus.publish-plugin") version "1.3.0"
}
```

As per the README? 
https://github.com/gradle-nexus/publish-plugin

Reply:
In which case it should just be in the plugins block with everything else.

Reply:
Plugin is to be conditionally applied as it should only be applied to the root project (not subprojects).